src/main/java/com/school/project/repository/AttendanceRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.school.project.repository;
+
+import com.school.project.model.entities.Attendance;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface AttendanceRepository extends JpaRepository<Attendance, Long> {
+
+    Optional<List<Attendance>> getAttendancesByLesson_GroupId(Long id);
```

Comment:
I'm wondering if this will return the correct Attendance list, usually you need to wreite @Query for this. Pls double check it

Reply:
@MichelRogov I checked it twice, this method gives us just what we need.